test/asdf_nu.bats

```diff
@@ -3,10 +3,11 @@
 load test_helpers
 
 setup() {
-  if [ "${SKIP_NUSHELL_TESTS}" = 'YES' ]; then
-    skip "skipping nushell tests"
-  fi
   cd $(dirname "$BATS_TEST_DIRNAME")
+
+  if ! command -v nu; then
+    skip "Nu is not installed"
+  fi
```

Comment:
Just curious: why did you move this `if` from the top of the `setup` function to the bottom?

Reply:
I believe I wanted to make it clear the the current directory was an invariant with respect to the test file, so moving it to the very top would accomplish that. I think it matches other test files as well